cmd/go-filecoin/daemon.go

```diff
@@ -95,7 +94,6 @@ func daemonRun(req *cmds.Request, re cmds.ResponseEmitter) error {
 		return errors.Wrap(err, "Bad block time passed")
 	}
 	opts = append(opts, node.BlockTime(blockTime))
-	opts = append(opts, node.ClockConfigOption(clock.NewSystemClock()))
```

Comment:
This is now done in the node builder since we need the genesis timestamp to init the chain clock.